azurelinuxagent/ga/monitor.py

```diff
@@ -193,3 +193,13 @@ def daemon(self):
             except Exception as e:
                 logger.warn("Failed to send events: {0}", e)
             time.sleep(60)
+
+    def add_sysinfo(self, event):
+        event_param_dict = {}  # key: TelemetryEventParam.name, val: idx of the EventParam in event.parameters
+        for i in range(len(event.parameters)):
```

Comment:
@jinhyunr "value" vs. "val" and "index" vs. "idx". In comments we must be clear...